sql/catalyst/src/main/scala-2.12/org/apache/spark/sql/catalyst/expressions/ExpressionSet.scala

```diff
@@ -108,12 +108,24 @@ class ExpressionSet protected(
     newSet
   }
 
+  override def ++(elems: GenTraversableOnce[Expression]): ExpressionSet = {
```

Comment:
Could you add a comment on the method about why we don't use the `SetLike.default` here? Thanks.

Reply:
thanks, added.